lang/en.json

```diff
@@ -144,8 +144,8 @@
         "or": "or",
         "compare": "Compare",
         "sign_up": "Sign Up",
-        "login": "Login",
-        "logout": "Logout",
+        "login": "Log in",
```

Comment:
Agree it depends on context. When it's a verb, it is two words. When it's a noun, it's a single word. So I disagree that when it's used to mean "present your credentials" as an action, then it's "login". When it's an action, it's a verb and is two words. You will see this behavior on major sites like Google and Twitter. Also, here's a reference: http://grammarist.com/spelling/log-in-login/

In all of the places that I can find in the theme, they are all used in the context of verbs.